src/azure-cli/azure/cli/command_modules/eventhubs/custom.py

```diff
@@ -68,7 +92,7 @@ def cli_namespace_update(cmd, client, instance, tags=None, sku=None, capacity=No
 
 
 def cli_namespace_list(cmd, client, resource_group_name=None):
-    if cmd.supported_api_version(min_api='2017-04-01'):
+    if cmd.supported_api_version(min_api='2018-01-01-preview'):
```

Comment:
It seems that there is no need to change this, if list() is already supported from 2017-04-01.

Reply:
list_by_resource_group() is supported in 2018-01-01 preview version